bats/tests/helpers/vm.bash

```diff
@@ -17,6 +17,21 @@ pkill_by_path() {
     fi
 }
 
+windows_cleanup() {
+    # Fixed in WSL 0.64.0, however below is a workaround for windows10
+    # https://github.com/microsoft/WSL/releases/tag/0.64.0
+    export WSL_UTF8=1
+    WSLENV="$WSLENV":WSL_UTF8
+    # match rancher-desktop and not rancher-desktop-data
+    if is_windows && wsl_exe -l -v | grep -Po '(?<=^| )rancher-desktop(?!-data(?=\b|$))'; then
```

Comment:
What is the reason for the complex condition? Just run all the commands inside the block with `run` so we ignore the error:

```suggestion
    if is_windows; then
```

Or if you really want to check if the distro exist, run a simple command in it:

```suggestion
    if is_windows && wsl.exe -d rancher-desktop true >/dev/null; then
```